experimental/Origami/src/deck_group.jl

```diff
@@ -0,0 +1,54 @@
+function in_deck_group(o::Origami, sigma::PermGroupElem)
+    h = horizontal_perm(o)
+    v = vertical_perm(o)
+    for i in 1:degree(o)
+        if (i^sigma)^h != (i^h)^sigma || (i^sigma)^v != (i^v)^sigma
+            return false
+        end
+    end
+    return true
+end
+
+function deck_group(o::Origami)
+    function candidate_for_deck(origami, j)
+        sheets_to_visit = 2:degree(origami)
+        sigma = [j]
+        append!(sigma, fill(-1, degree(origami) - 1))
```

Comment:
One less allocation:
```suggestion
        sigma = fill(-1, degree(origami))
        sigma[1] = j
```